WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1790,13 +1780,20 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
                     break;
                 case RequestCodes.VIDEO_LIBRARY:
                     Uri videoUri = data.getData();
-                    fetchMedia(Arrays.asList(videoUri));
```

Comment:
It seems that it's not necessary since it's wrapped inside an `if` condition that checks `data != null`. Anyway I've added this check on both video library and media library branches.